firmware/Cargo.toml

```
@@ -33,7 +34,10 @@ default = ["mcu-esp32c3", "imu-mpu6050", "log-rtt"]
 
 # todo: actually support another microcontroller
 # Supported microcontrollers
-mcu-esp32c3 = ["defmt_esp_println?/esp32c3"]
+mcu-esp32c3 = ["defmt_esp_println?/esp32c3", "net-wifi", "esp-backtrace/esp32c3"]
```

Mcu category should not enable other categories, but only control platform-specific dependencies.
In this case we would do something like `esp-wifi?/esp32c3` but we don't even need to do do that since esp-wifi already covers all our current `mcu-*`. When we add support for nrf though, this will get a bit more complicated.

For now it is sufficient to not specify `esp-wifi` here
```suggestion
mcu-esp32c3 = ["defmt_esp_println?/esp32c3", "esp-backtrace/esp32c3"]
```